hpat/tests/test_series.py

```diff
@@ -1622,6 +1618,39 @@ def test_impl():
         hpat_func = hpat.jit(test_impl)
         np.testing.assert_array_equal(hpat_func(), test_impl())
 
+    def test_series_take_index_default(self):
+        def pyfunc():
+            series = pd.Series([1.0, 13.0, 9.0, -1.0, 7.0])
+            indices = [1, 3]
```

Comment:
Could you please tell why we don't want to test unboxing stage? I mean why don't we pass `series` and `indices` to the function as arguments?

Reply:
Testing of the list boxing functionality is the purpose of other specialized tests (in general, in good test system). But at current stage it might be better to have both variants here. I changed tests to localize an issue in development.

Reply:
I would vote for having functionality tests for both, in JITted and out of JITted regions